### PR TITLE
UAR-1594 add temp logging for debugging

### PR DESCRIPTION
**Pull request by dlloyd1-ch**
### JIRA link
https://companieshouse.atlassian.net/browse/UAR-1594

### Change description
Added some additional logging statements to make it clear if private BO data could not be retrieved for some reason.
Fixed indentation in test file.

### Work checklist

- [ ] Tests added where applicable
- [ ] UI changes meet accessibility criteria

### Merge instructions

We are committed to keeping commit history clean, consistent and linear. To achieve this, this commit should be structured as follows:

```
<type>[optional scope]: <description>
```

and contain the following structural elements:

- fix: a commit that patches a bug in your codebase (this correlates with PATCH in semantic versioning),
- feat: a commit that introduces a new feature to the codebase (this correlates with MINOR in semantic versioning),
- BREAKING CHANGE: a commit that has a footer `BREAKING CHANGE:` introduces a breaking API change (correlating with MAJOR in semantic versioning). A BREAKING CHANGE can be part of commits of any type,
- types other than `fix:` and `feat:` are allowed, for example `build:`, `chore:`, `ci:`, `docs:`, `style:`, `refactor:`, `perf:`, `test:`, and others,
- footers other than `BREAKING CHANGE: <description>` may be provided.

### CODE REVIEW

File: src/utils/update/fetch.beneficial.owners.private.data.ts [hunk 1]
```diff
@@ -4,7 +4,7 @@ import { ApplicationData } from "../../model";
 import { mapCorporateOrGovernmentBOPrivateData, mapIndividualBOPrivateData } from "./psc.to.beneficial.owner.type.mapper";
 
 export const fetchBeneficialOwnersPrivateData = async (appData: ApplicationData, req) => {
-
+  logger.info(`******** Fetching private beneficial owners data ********`);
```
Comment (mwestacott) [inline src/utils/update/fetch.beneficial.owners.private.data.ts:7]: Do we need all the ******s in these messages? Suggest we remove them.

File: src/utils/update/fetch.beneficial.owners.private.data.ts [hunk 2]
```diff
@@ -26,6 +26,7 @@ export const fetchBeneficialOwnersPrivateData = async (appData: ApplicationData,
 
 const mapBeneficialOwnersPrivateData = (boPrivateData, appData: ApplicationData) => {
   const mapIndividualOwners = beneficialOwner => {
+    logger.info(`************* beneficialOwner.ch_reference: ${beneficialOwner.ch_reference} ***************`);
```
Comment (mwestacott) [inline src/utils/update/fetch.beneficial.owners.private.data.ts:29]: For consistency, suggest adding a similar line in the Gov/Corporate function, around line 35.

File: src/utils/update/fetch.beneficial.owners.private.data.ts (hunk 1 above)
Comment (dlloyd1-ch) [reply src/utils/update/fetch.beneficial.owners.private.data.ts:7]: They're only temp messages, but happy to remove the asterisks

**Review by mwestacott — COMMENTED** [review]
Some additional logging within the `mapIndividualBOPrivateData()` function might be helpful too. What do you think? It's in the `psc.to.beneficial.owner.type.mapper.ts` file and called when mapping the address.

File: src/utils/update/fetch.beneficial.owners.private.data.ts (hunk 1 above)
Comment (mwestacott) [reply src/utils/update/fetch.beneficial.owners.private.data.ts:7]: Fine if intention is to remove them, but then we should have a TODO comment stating this. I think just remove the *s. Up to you though.

**Comment by dlloyd1-ch** [comment]
> Some additional logging within the `mapIndividualBOPrivateData()` function might be helpful too. What do you think? It's in the `psc.to.beneficial.owner.type.mapper.ts` file and called when mapping the address.

I've already added logging to this function.

File: src/utils/update/fetch.beneficial.owners.private.data.ts (hunk 1 above)
Comment (dlloyd1-ch) [reply src/utils/update/fetch.beneficial.owners.private.data.ts]: Removed the asterisks.

**Pull request merged**